lib/chef/resource/package.rb

```diff
@@ -31,14 +31,13 @@ class Package < Chef::Resource
 
       def initialize(name, run_context=nil)
         super
-        @candidate_version = nil
-        @options = nil
-        @package_name = name
-        @response_file = nil
-        @response_file_variables = Hash.new
-        @source = nil
-        @version = nil
-        @timeout = nil
+        options                 nil
+        package_name            name
+        response_file           nil
+        response_file_variables Hash.new
+        source                  nil
+        version                 nil
+        timeout                 nil
```

Comment:
That makes pragmatic sense and still gives us the benefit of initializing our state at creation time and documents the properties of the object in a single place, so :+1: